runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -402,12 +451,13 @@ class LayoutFFITypeHelpers
 	/* @brief This wrapper function invokes parseStruct() to determine
 	 * the AGGREGATE subtype of the specified struct.
 	 *
-	 * @param structSig[in] A pointer to the specified struct signature string
+	 * @param structSig[in/out] A pointer to the specified struct signature string,
+	 * which points to the end of the signature string upon return.
```

Comment:
This is not an "out" parameter, so there's nothing to describe for callers.

Reply:
Removed.